src/engine/SCons/Util.py

```diff
@@ -63,7 +60,7 @@
 FunctionType    = types.FunctionType
 
 try:
-    unicode
+    _ = type(unicode)
```

Comment:
Why? (Curious)

Reply:
just to be running a "real" Python command which still generates a NameError on Py3.  I don't have any particular attachment to this, PyCharm grumped at the original, but don't mind going back, this wasn't an important part of the change.
Py2:
```
>>> unicode
<type 'unicode'>
>>> _ = type(unicode)
>>> 
```
Py3:
```
>>> unicode
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'unicode' is not defined
>>> _ = type(unicode)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'unicode' is not defined
>>> 
```

Reply:
Interesting.
O.k. leave it as is.

At some point I'd like to add pylint  or equivalent to our CI..